csharp/extractor/Semmle.Extraction/Entity.cs

```diff
@@ -79,6 +79,8 @@ public interface ICachedEntity : IEntity
         void Populate();
 
         bool NeedsPopulation { get; }
+
+        object UnderlyingObject { get; }
```

Comment:
why added?

Reply:
It was quite useful for debugging. I'm minded to keep it in if that's ok?